.github/workflows/ci.yaml

```diff
@@ -26,16 +27,17 @@ jobs:
             CATKIN_LINT: true
     env:
       DOCKER_IMAGE: moveit/moveit:${{ matrix.env.IMAGE }}
-      UPSTREAM_WORKSPACE: moveit.rosinstall
-      # Pull any updates to the upstream workspace (since after restoring it from cache)
+      UPSTREAM_WORKSPACE: .github/workflows/upstream.rosinstall
+      DOWNSTREAM_WORKSPACE: .github/workflows/downstream.rosinstall
+      # Pull any updates to the upstream workspace (after restoring it from cache)
       AFTER_SETUP_UPSTREAM_WORKSPACE: vcs pull $BASEDIR/upstream_ws/src
+      AFTER_SETUP_DOWNSTREAM_WORKSPACE: vcs pull $BASEDIR/downstream_ws/src
       AFTER_RUN_TARGET_TEST: ${{ matrix.env.CCOV && './.ci.prepare_codecov' || '' }}
       TARGET_CMAKE_ARGS: >
         -DCMAKE_BUILD_TYPE=${{ matrix.env.CCOV && 'RelWithDebInfo' || 'Release'}}
         ${{ matrix.env.CCOV && '-DCMAKE_CXX_FLAGS="--coverage" --no-warn-unused-cli' || '' }}
       CCACHE_DIR: "${{ github.workspace }}/.ccache"
       BASEDIR: ${{ github.workspace }}/.work
-      CACHE_PREFIX: "${{ matrix.env.IMAGE }}${{ matrix.env.CCOV && '-ccov' || '' }}"
```

Comment:
Did you mean to remove this variable?

Reply:
Yes, it's implemented locally for each caching step now.